src/Dimension.ts

```diff
@@ -150,7 +171,7 @@ export default class Dimension {
       throw new Error('Dimensions array size mismatch...')
     }
     // Check for order
-    _.range(dims1.length).forEach((i) => {
+    for (let i = 0; i < dims1.length; i++) {
```

Comment:
This is purely for performance, I guess.

Reply:
Yes, just to avoid unnecessary function calls and temporary arrays. I'm not a fan of `_.range` and `forEach` in general, as those inhibit loop optimizations in the optimizing compiler (due to extra complexity from allocations and function calls). There is absolutely nothing wrong with a for loop :)